src/Injectable.ts

```diff
@@ -100,6 +100,27 @@ export function Injectable(
   return factory;
 }
 
+/**
+ * Creates an Injectable factory function for an InjectableClass.
+ *
+ * @param token Token identifying the Service.
```

Comment:
Let's provide an example how this can be used similarly how we do that for other exported stuff here